### ISSUE TITLE
 Visual rendering freezes with mod enabled, audio continues

### ISSUE THREAD

**Issue by Alessio12345678** [Wynntils/Wynntils#2802]
When using this mod, the visual rendering of the game freezes at some point during gameplay. Despite the visual freeze, the audio continues to play normally. The game becomes unresponsive, and I am unable to perform any actions. This issue only occurs when this mod is enabled; the game functions properly without it. I have tested this multiple times to confirm the mod is the cause of the problem.

Additional Information:

- Minecraft version: 1.21.1

- Wynntils version: latest

Please let me know if you need any additional information to help troubleshoot this issue.

**Comment by magicus** [comment]
Do you have any other mods installed? If so, please try to reproduce the issue with only Wynntils installed.

**Comment by Alessio12345678** [comment]
No, I do not have any other mods installed. I have attempted to install Sodium to potentially resolve the issue, but the same problem persists, just with a delay.

**Comment by magicus** [comment]
That audio continues to play might be just buffering. 

Does the game eventually "unfreeze", or do you have to kill Minecraft?

Can you provide the Minecraft log?

**Comment by Alessio12345678** [comment]
No, it doesn't unfreeze and so i have to kill Minecraft.
Here is the log you asked for:
[latest.log](https://github.com/user-attachments/files/16829050/latest.log)

**Comment by magicus** [comment]
The log is utterly unhelpful:
```
[23:28:02] [Render thread/INFO]: Message Queued: TextRenderTask[text=StyledText{'§3You have §l1§r§3 unused ability points'}, setting=TextRenderSetting[maxWidth=0.0, customColor=#ffffffff, horizontalAlignment=LEFT, verticalAlignment=TOP, shadow=NORMAL]]
[23:28:09] [Render thread/INFO]: Stopping!
[23:28:09] [Render thread/INFO]: Preloaded pack is up-to-date, no-op.
[23:28:09] [Render thread/INFO]: Stopping worker threads
[23:28:09] [Thread-3/INFO]: Successfully saved translation cache!
```

I'm not sure what the "Stopping!" is about, but from the saved translation cache it at least seem that Minecraft did try to shut down gracefully to some degree.

**Comment by ShadowCat117** [comment]
Unless this issue has continued on recent versions I will close it for now as there have not been other reports of this issue and a lot has changed in the time since this was posted and may have "unintentionally" been fixed
